common/src/messaging/crossbeam_messaging.rs

```diff
@@ -0,0 +1,75 @@
+use std::fmt::Debug;
+
+use async_trait::async_trait;
+
+use super::{AsyncReceiver, AsyncSender};
+
+#[derive(Debug)]
+pub enum CrossbeamSenderError {
+    ChannelDisconnected,
+}
+
+impl std::error::Error for CrossbeamSenderError {}
+
+impl std::fmt::Display for CrossbeamSenderError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            CrossbeamSenderError::ChannelDisconnected => write!(f, "Channel was disconnected"),
+        }
+    }
+}
+
+#[derive(Debug)]
+pub enum CrossbeamReceiverError {
```

Comment:
why not import / use the couple : thiserror / anyhow ?

Reply:
The initial thought was to reduce the deps list, but it does indeed make sense